prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var declareFirstElement = 'The first element of the array is';
+var firstElement = 'html';
+console.log(declareFirstElement, firstElement);
+var declareThirdElement = 'The 3rd element of the array is';
+var thirdElement = 'javascript';
```

Comment:
Here you should be using the `languages` array to get the 3rd item in the array. The 3rd item in the array is at index `2` since the indexes start at `0`